ModComponent/API/Components/ModPowderComponent.ModPowderType.cs

```diff
@@ -1,13 +1,11 @@
-﻿extern alias Hinterland;
+﻿using Il2Cpp;
 
 namespace ModComponent.API.Components;
 
 public partial class ModPowderComponent
 {
 	public enum ModPowderType
 	{
-		Gunpowder,
-		Salt,
-		Yeast
+		Gunpowder
 	}
```

Comment:
The Hinterland enums have been shown to be stable over large time periods. I think I want most of the "mod" enums removed. Any "mod" enums that aren't direct copies can stay.